downstream/modules/platform/proc-upgrade-controller-hub-eda-unified-services.adoc

```diff
@@ -3,17 +3,17 @@
 :_mod-docs-content-type: PROCEDURE
 
 [id="upgrade-controller-hub-eda-unified-ui-services_{context}"]
-= Using Migration path for 2.4 services with 2.5 services
+= Using Migration path for 2.4 services with 2.6 services
 
-If you installed {PlatformNameShort} 2.5 to use {EDAName} in a supported scenario, you can upgrade your {PlatformNameShort} 2.4 {ControllerName} and {HubName} to {PlatformNameShort} 2.5 by following the procedure in this section. 
+If you installed {PlatformNameShort} {PlatformVers} to use {EDAName} in a supported scenario, you can upgrade your {PlatformNameShort} 2.4 {ControllerName} and {HubName} to {PlatformNameShort} {PlatformVers} by following the procedure in this section. 
 
 .Prerequisites
 
-* An inventory from 2.4 for {ControllerName} and {HubName} and a 2.5 inventory for unified UI ({Gateway}) and {EDAName}. You must run upgrades on 2.4 services (using the inventory file to specify only {ControllerName} and {HubName} VMs) to get them to the initial version of {PlatformNameShort} 2.5 first. When all the services are at the same version, run an upgrade (using a complete inventory file) on all the services to go to the latest version of {PlatformNameShort} 2.5.
+* An inventory from 2.4 for {ControllerName} and {HubName} and a 2.6 inventory for unified UI ({Gateway}) and {EDAName}. You must run upgrades on 2.4 services (using the inventory file to specify only {ControllerName} and {HubName} VMs) to get them to the initial version of {PlatformNameShort} {PlatformVers} first. When all the services are at the same version, run an upgrade (using a complete inventory file) on all the services to go to the latest version of {PlatformNameShort} {PlatformVers}.
```

Comment:
Should we apply {PlatformVers} to each instance of 2.6? (Suggesting consistency either way.)